packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -93,6 +94,44 @@ To auto-complete potential request type values for `"requestTypeFilter"`, press
 
+## ISV Customer Debugger
+
+The ISV debugging capability covers a gap in what you can do with the Apex Debugger. As an ISV, you can debug your own code. As a subscriber, you can debug your own code. However, because of the protections against seeing managed code, subscribers can’t debug ISV code in their orgs.
+
+With the ISV Customer Debugger, an ISV can work with a subscriber to debug issues specific to the subscriber’s org. An ISV can reproduce issues in the specific environment, so problems can be diagnosed more quickly, facilitating faster fixes for the ISV’s software offering. You can debug only sandbox orgs.
+
+### Set Up the ISV Customer Debugger
+
+The ISV Customer Debugger is part of the `salesforcedx-vscode-apex-debugger` extension, so you don’t need to install anything other than this extension and its prerequisites. Log in to your subscriber’s sandbox via your License Management Org (LMO) and copy a URL from Setup. Then, create a debugging project and start a session from VS Code.
+
+1. Log in to your subscriber’s sandbox. If you’re not familiar with this process, see the _ISVforce Guide_. For information on how to obtain login access to your subscriber’s org, see [Request Login Access from a Customer](https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/lma_requesting_login_access.htm). For information on how to log in via the Subscriber Support Console, see [Logging In to Subscriber Orgs](https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/lma_logging_in_to_sub_org.htm). You can debug only sandbox orgs.
+1. In your subscriber’s org, from Setup, enter **Apex Debugger** in the Quick Find box, then click **Apex Debugger**.
+1. Click **Start Partner Debugging Session**.
+1. In the Using Salesforce Extensions for VS Code section, to copy the `forceide://` URL, click **Copy to Clipboard**.
+1. In VS Code, press Cmd+Shift+P (macOS) or Ctrl+Shift+P (Windows or Linux) to open the command palette, then run **SFDX: Create and Set Up Project for ISV Debugging**.
+1. When directed to, paste the `forceide://` URL into the prompt and then press Enter.
+1. When directed to, enter a name for your debugging project and then press Enter.
+1. Choose a location to store the project, and then click **Create Project**.
+1. Wait for the project generation process to finish. VS Code creates a Salesforce DX project that contains your packaged metadata, your subscriber's metadata, and skeleton classes for other packages in the org. VS Code also creates a launch configuration (`launch.json` file) for the project. When the project is ready, VS Code opens it for you in a new window.
+1. In the new window, from the Explorer view, open an Apex class or trigger that you want to set breakpoints in.
+1. Switch to the Debug view.
+1. To set a breakpoint, click the gutter to the left of the line numbers.
```

Comment:
117 and 118 seems to be reversed, i.e. I would set breakpoint first and then switch to Debug view.